dags/bqetl_monitoring.py

```diff
@@ -110,8 +110,9 @@
         project_id="moz-fx-data-shared-prod",
         owner="ascholtz@mozilla.com",
         email=["ascholtz@mozilla.com"],
-        date_partition_parameter="submission_date",
+        date_partition_parameter=None,
         depends_on_past=False,
+        parameters=["submission_date:DATE:{{ds}}"],
```

Comment:
We cannot use `date_partition_parameter` here since there is an `INSERT INTO` statement as part of the script instead for writing selected data into a partition.